crates/bevy_text/src/text.rs

```diff
@@ -170,3 +173,26 @@ impl Default for TextStyle {
         }
     }
 }
+
+/// Determines how lines will be broken when preventing text from running out of bounds.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, Reflect, Serialize, Deserialize)]
+#[reflect(Serialize, Deserialize)]
+pub enum BreakLineOn {
+    /// Uses the [Unicode Line Breaking Algorithm](https://www.unicode.org/reports/tr14/).<br/>
+    /// Lines will be broken up at the nearest suitable word boundary, usually a space.<br/>
+    /// This behaviour suits most cases, as it keeps words intact across linebreaks.<br/>
+    WordBoundary,
+    /// Lines will be broken without discrimination on any character that would leave bounds.<br/>
+    /// This is closer to the behaviour one might expect from text in a terminal. <br/>
+    /// However it may lead to words being broken up across linebreaks<br />
```

Comment:
Nit: (non-blocking) For docs, I think it's better to not use `<br/>`, but rather use an empty line.